server/odc-service/src/main/java/com/oceanbase/odc/service/connection/ConnectionService.java

```diff
@@ -488,69 +522,105 @@ public PageAndStats<ConnectionConfig> list(@Valid QueryConnectionParams params,
         return PageAndStats.of(connections, stats);
     }
 
-    @Transactional(rollbackFor = Exception.class)
```

Comment:
why don't we use `@Transactional`?

Reply:
When update a `connectionConfig`, we will submit a task to sync databases. If we use `@Transactional`, the transaction will not commit until the method is returned. So when the task start to sync databases, the `connectionConfig` which the task get from metaDB may be dirty data.